executor/cook/progress.py

```diff
@@ -290,7 +298,7 @@ class ProgressTracker(object):
     """Helper class to track progress messages from the specified location."""
 
     def __init__(self, config, stop_signal, task_completed_signal, counter, progress_updater,
-                 progress_termination_signal, location, location_tag):
+                 progress_termination_signal, location, location_tag, os_error_handler):
```

Comment:
Same here

Reply:
This is in the constructor and one of the calls will start a thread where we need to guard the executing function with the handler.